src/frontends/tensorflow_lite/src/graph_iterator_flatbuffer.cpp

```diff
@@ -87,7 +87,11 @@ std::shared_ptr<DecoderBase> GraphIteratorFlatBuffer::get_decoder() const {
         auto op_codes = m_model->operator_codes();
         auto operator_code = (*op_codes)[node->opcode_index()];
         std::string type;
-        if (operator_code->deprecated_builtin_code() <
+        if (operator_code->deprecated_builtin_code() != operator_code->builtin_code()) {
+            // if the two enum values are different, take the non-zero one which means specified.
+            type = tflite::EnumNamesBuiltinOperator()[operator_code->deprecated_builtin_code() ? \
+               operator_code->deprecated_builtin_code() : operator_code->builtin_code()];
+        } else if (operator_code->deprecated_builtin_code() <
             tflite::BuiltinOperator::BuiltinOperator_PLACEHOLDER_FOR_GREATER_OP_CODES) {
             type = tflite::EnumNamesBuiltinOperator()[operator_code->deprecated_builtin_code()];
         } else {
```

Comment:
you logic is equivalent to just:

```
            type = tflite::EnumNamesBuiltinOperator()[operator_code->deprecated_builtin_code() ? \
               operator_code->deprecated_builtin_code() : operator_code->builtin_code()];
```

so no need to have `if-else` branches

Reply:
Thanks, removing all the "if -- else if -- else" and using something like below?
```
            type = tflite::EnumNamesBuiltinOperator()[operator_code->deprecated_builtin_code() &&\
               (operator_code->deprecated_builtin_code() < \
                 tflite::BuiltinOperator::BuiltinOperator_PLACEHOLDER_FOR_GREATER_OP_CODES) ? \
               operator_code->deprecated_builtin_code() : operator_code->builtin_code()];
```